worker/codalabworker/local_run/local_run_manager.py

```diff
@@ -360,3 +359,22 @@ def memory_bytes(self):
         except ValueError:
             # Fallback to sysctl when os.sysconf('SC_PHYS_PAGES') fails on OS X
             return int(check_output(['sysctl', '-n', 'hw.memsize']).strip())
+
+    @property
+    def available_disk_space(self):
+        """
+        Available disk space by bytes of this RunManager.
+        """
+        p = Popen(["df", "./"], stdout=PIPE)
+        output, error = p.communicate()
+        # Return -1 when there is an error.
+        if error:
+            logger.info(error.strip())
+            return -1
+
+        if output:
+            lines = output.decode().split("\n")
+            index = lines[0].split().index("Available")
```

Comment:
What happens if this code throws an exception for whatever reason?  Is it caught somewhere?  Just make sure this doesn't crash the worker.